src/messaging/discord.ts

```diff
@@ -7,6 +7,9 @@ import {RawUserData} from 'discord.js/typings/rawDataTypes';
 
 const {notifyGroup, webhooks, notifyGroupSeries} = config.notifications.discord;
 const {pollInterval, responseTimeout, token, userId} = config.captchaHandler;
+const clientOptions: Discord.ClientOptions = {
+  intents: new Discord.Intents(),
```

Comment:
I am not quite certain which Discord gateway intents are here necessary. `Intents.FLAGS.DIRECT_MESSAGES` seems to be a very suitable choice. But when I tested this, it was enough if I didn't specify any intents at all.
Discord.js  documentation also contains the following [warning](https://discordjs.guide/popular-topics/intents.html#enabling-intents):
> Note that discord.js relies heavily on caching to provide its functionality. Some methods that seem unrelated might stop working if certain events do not arrive.

Reply:
Yeah, I kind of haphazardly updated to v13 and changed a few types to make it compile. Thanks for fishing through this.